scripts/lint_translations.clj

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bb
+
+(ns lint-translations
+  (:require [babashka.pods :as pods]))
+
+(pods/load-pod 'clj-kondo/clj-kondo "2023.09.07")
+(require '[pod.borkdude.clj-kondo :as kondo])
+(require '[cheshire.core :as json])
+(require '[clojure.set :as set])
+
+(defn- analyze-code
+  [paths]
+  (kondo/run!
+   {:lint   paths
+    :config {:output {:analysis {:keywords true}}}}))
+
+(defn- safe-name
+  [x]
+  (when x (name x)))
+
+(defn- ->keyword
+  [analysis-keyword]
+  (keyword (safe-name (:ns analysis-keyword)) (:name analysis-keyword)))
+
+(defn- report-issues
+  [incorrect-usages]
+  (doseq [incorrect-usage incorrect-usages]
+    (->> incorrect-usage
+         ((juxt :filename :row :reason ->keyword))
+         (apply format "%s:%s %s %s")
+         println)))
+
+(defn- extract-translation-keys
+  [file]
+  (-> file slurp json/parse-string keys))
+
+(def ^:private translation-file "translations/en.json")
+
+;; (def unused-warning (format "Unused Translation Key in %s:" translation-file))
```

Comment:
can we delete commented code instead 🙏

Reply:
hmm, all the comments here were left half intentionally for the case we ever want to distinguish between unused translations and usage of non-namespaced translation keys...

As I pointed out in the description this is not always generally possible (as there would be too many false postivies for non-namespaced translation keys), and certainly not appropriate at this stage, but could be appropriate in the future once all actual translation keys in the code are namespaced, to catch future non-namespaced translation keys.

Reply:
I am happy to delete the commented out code for now though, as it should fairly straightforward to re-create this functionality in the future if and when needed.

Reply:
yeah, in my perspective I think it's better to have this tracked in a separate issue or in this pr description/self comments so that if we ever want to revisit we can return to the pr and see any extra details that might be relevant about this work.

Reply:
updated description